mail/models.py

```diff
@@ -0,0 +1,19 @@
+"""
+Models for mail
+"""
+from django.contrib.auth.models import User
+from django.db import models
+
+from financialaid.models import TimestampedModel, FinancialAid
+
+
+class FinancialAidEmailAudit(TimestampedModel):
+    """
+    Audit table for the Financial Aid
+    """
+    acting_user = models.ForeignKey(User, null=False)
+    financial_aid = models.ForeignKey(FinancialAid, null=True, on_delete=models.SET_NULL)
+    to_email = models.CharField(null=False, max_length=250)
```

Comment:
Can we use `TextField` here and below instead? AFAIK there is no performance penalty for using `TextField` when working with postgres